rotkehlchen/db/upgrades/v40_v41.py

```diff
@@ -22,15 +36,41 @@ def _add_cache_table(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_cache_table')
 
 
+def _get_unrelated_mappings(read_cursor: 'DBCursor') -> dict:
+    """Get the unrelated key-value pairs from `settings`"""
+    log.debug('Enter _get_unrelated_mappings')
+    settings = read_cursor.execute('SELECT name, value FROM settings').fetchall()
```

Comment:
don't do a fetchall if you are going to iterate after. Fetchall is already a loop in the cursor and then you loop it again